modules/project-docs/pages/compatibility.adoc

```diff
@@ -63,9 +63,10 @@ Other versions may work -- _but they are not tested and they are not supported_.
 
 The following JDK releases are supported at time of 3.4 SDK release (October 2022):
```

Comment:
Is 21 supported right back across 3.4.x releases?

Reply:
@dnault ?

Reply:
I looked into this - will reword tomorrow to say only latest version for JDK 21.

Reply:
Added to line 66 (below).